ios/CameraView+Focus.swift

```diff
@@ -74,24 +74,64 @@ extension CameraView {
       }
 
       // in {0..1} system
-      let normalizedPoint = captureDevicePointConverted(fromLayerPoint: point)
+      var normalizedPoint: CGPoint
+      if let previewView = previewView as? PreviewView {
+        // previewView is of type PreviewView can use the built in captureDevicePointConverted
+        normalizedPoint = previewView.videoPreviewLayer.captureDevicePointConverted(fromLayerPoint: point)
+      } else {
+        normalizedPoint = captureDevicePointConverted(fromLayerPoint: point)
+      }
 
       do {
         try device.lockForConfiguration()
 
         device.focusPointOfInterest = normalizedPoint
-        device.focusMode = .continuousAutoFocus
+        device.focusMode = .autoFocus
 
         if device.isExposurePointOfInterestSupported {
           device.exposurePointOfInterest = normalizedPoint
-          device.exposureMode = .continuousAutoExposure
+          device.exposureMode = .autoExpose
         }
 
+        // Enable subject area change monitoring
+        device.isSubjectAreaChangeMonitoringEnabled = true
+
+        // Remove any existing observer for subject area change notifications
+        NotificationCenter.default.removeObserver(self, name: NSNotification.Name.AVCaptureDeviceSubjectAreaDidChange, object: nil)
+
+        // Register observer for subject area change notifications
+        NotificationCenter.default.addObserver(self, selector: #selector(subjectAreaDidChange), name: NSNotification.Name.AVCaptureDeviceSubjectAreaDidChange, object: nil)
+
         device.unlockForConfiguration()
         return nil
       } catch {
         throw CameraError.device(DeviceError.configureError)
       }
     }
   }
+
+  @objc func subjectAreaDidChange(notification: NSNotification) {
```

Comment:
Question: Does this get called immediately after it successfully focused on the given point?

Reply:
No it gets called when the subject area changes. This means that if you tap to focus on a "subject" the focus gets locked on that object, but of there is movement in that area then the callback happens and the focus and exposure are set back to `continuous` mode and will adjust automatically (removing the lock that the focus event enabled with `autoFocus` and `autoExpose`